share/module_check_a_mundo.F

```diff
@@ -569,6 +570,27 @@ END FUNCTION bep_bem_nbui_max
       END DO
 
 #if (EM_CORE == 1)
+!--------------------------------------------------------------------------------------------------
+! Input tables must exist in running directory for fast bin microphysics scheme (mp_physics = 30)
+!--------------------------------------------------------------------------------------------------
+      IF (  model_config_rec % mp_physics(1) .EQ. FAST_KHAIN_LYNN_SHPUND   ) THEN
+           INQUIRE(FILE='./SBM_input_33', EXIST=fsbm_table1_exists)
+           IF (.not.fsbm_table1_exists ) THEN
```

Comment:
@smileMchen 
Ming,
Follow the indenting convention that seems to be in use (looks like 3 spaces).

Reply:
@davegill 
Dave, the indent has been changed to keep consistent with codes below.